snap/epoch_test.go

```diff
@@ -230,6 +230,20 @@ func (s *epochSuite) TestE(c *check.C) {
 	}
 }
 
+func (s *epochSuite) TestUnset(c *check.C) {
+	for _, e := range []*snap.Epoch{nil, {}} {
+		c.Check(e.Unset(), check.Equals, true, check.Commentf("%#v", e))
+	}
+	for _, e := range []*snap.Epoch{
+		{Read: []uint32{0}, Write: []uint32{0}},
```

Comment:
is it a problem that Unset() is serialized as"0",  but "0" is not Unset ?

Reply:
it might be a bit annoying. Not too much, I think, but a little. We could deserialise 0 as unset everywhere? if that'd help.

Reply:
seems more in line with what we do?  either way I was more worried if it could be a bug magnet

Reply:
in a followup I'll change the fromString and fromStructured so that anything that would serialise back to "0" is stored as unset